native/cocos/renderer/pipeline/GlobalDescriptorSetManager.cpp

```diff
@@ -111,11 +112,12 @@ void GlobalDSManager::update() {
     }
 }
 
-gfx::DescriptorSet *GlobalDSManager::getOrCreateDescriptorSet(uint32_t idx) {
+gfx::DescriptorSet *GlobalDSManager::getOrCreateDescriptorSet(const scene::Light *light) {
+    CC_ASSERT(light);
     // The global descriptorSet is managed by the pipeline and binds the buffer
-    if (_descriptorSetMap.count(idx) <= 0 || (_descriptorSetMap.at(idx) == nullptr)) {
+    if (_descriptorSetMap.count(light) <= 0 || (_descriptorSetMap.at(light) == nullptr)) {
```

Comment:
`_descriptorSetMap.count(light) <= 0` -> `_descriptorSetMap.count(light) == 0` as return value will not less than 0.

Reply:
sure